wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -652,6 +653,48 @@ <h2>Dropdown buttons</h2>
+
+                    {% dialog icon_name="doc-full-inverse" title="Simple dialog" subtitle="This is as simple as it gets 😀" %}
+                        <p class="w-base-text">This is an example of a simple dialog with an icon_name, title and subtitle passed to the dialog tag</p>
+                    {% enddialog %}
+                </div>
+                <div>
+                    {% dialog_toggle class_name='button button-primary' dialog_title='Dialog with info' text='Dialog with info' %}
+
+                    {% dialog icon_name="globe" title="Dialog with info" subtitle="This is a testing subtitle" message_status="info" message_heading="Here is some info on the thing" message_description="This is a subtext for the message" %}
+                        <p class="w-base-text">This dialog message was generated by passing message_status=info as well as message_heading and message_description to the dialog template tag</p>
+                    {% enddialog %}
+                </div>
+                <div>
+                    {% dialog_toggle class_name='button button-primary' dialog_title='Dialog with success' text='Dialog with success' %}
+
+                    {% dialog icon_name="globe" title="Dialog with success" subtitle="This is a testing subtitle" message_status="success" message_heading="Success! You've done the thing"  %}
+                        <p class="w-base-text">This dialog message was generated by passing message_status=success as well as message_heading to the dialog template tag</p>
+                    {% enddialog %}
+                </div>
+                <div>
+                    {% dialog_toggle class_name='button button-primary' dialog_title='Dialog with warning' text='Dialog with warning' %}
+
+                    {% dialog icon_name="globe" title="Dialog with warning" subtitle="This is a testing subtitle" message_status="warning" message_heading="There was an issue with the thing" message_description="This is a subtext for the message" %}
+                        <p class="w-base-text">This dialog message was generated by passing message_status=warning as well as message_heading and message_description to the dialog template tag</p>
+                    {% enddialog %}
+                </div>
+                <div>
+                    {% dialog_toggle class_name='button button-primary' dialog_title='Dialog with critical error' text='Dialog with error' %}
+
+                    {% dialog icon_name="globe" title="Dialog with critical error" subtitle="This is a testing subtitle" message_status="critical" message_heading="There was an issue with the thing" message_description="This is a subtext for the message" %}
+                        <p class="w-base-text">This dialog message was generated by passing message_status=critical as well as message_heading and message_description to the dialog template tag</p>
+                    {% enddialog %}
+                </div>
+            </div>
+        </section>
+
```

Comment:
Are we adding stories as part of this PR also?

Reply:
I think this will be part of a separate PR unfortunately :/

Reply:
Good call. Just wanted to check it was on the radar.